diff --git a/node_modules/docusaurus-plugin-glossary/dist/index.cjs b/node_modules/docusaurus-plugin-glossary/dist/index.cjs
index f956a7e..61e3665 100644
--- a/node_modules/docusaurus-plugin-glossary/dist/index.cjs
+++ b/node_modules/docusaurus-plugin-glossary/dist/index.cjs
@@ -557,9 +557,7 @@ ${errorList}`;
 }
 
 // src/index.ts
-var import_meta = {};
-var currentFilePath = (0, import_url.fileURLToPath)(import_meta.url);
-var currentDir = import_path2.default.dirname(currentFilePath);
+var currentDir = __dirname;
 (0, import_validate_peer_dependencies.default)(currentDir);
 function glossaryPlugin(context, options = {}) {
   const { glossaryPath = "glossary/glossary.json", routePath = "/glossary" } = options;
